frappe/desk/search.py

```diff
@@ -223,12 +236,13 @@ def search_widget(
 				strict=False,
 			)
 
-			if doctype in translated_search_doctypes:
+			if doctype in translated_doctypes:
 				# Filtering the values array so that query is included in very element
 				values = (
 					v
 					for v in values
 					if re.search(f"{re.escape(txt)}.*", _(v.name if as_dict else v[0]), re.IGNORECASE)
+					or re.search(f"{_(re.escape(txt))}.*", _(v.name if as_dict else v[0]), re.IGNORECASE)
```

Comment:
Not sure if this is helpful / needed. What's the intended effect?

Reply:
- This only searches for translated words
- let's take an example:
- if I want to search for `Nichtbinär` viz `Genderqueer`, I can search with both Nichtbinär as well as Genderqueer, and in both cases, the system should match the word, which is why searching using both the original word and translated would increase the matches

Reply:
Somewhere there's still a problem, because `"Nichtbin"` will produce a match, while `"Genderq"` won't. If I'm typing untranslated, I had to type the exact name (`"Genderqueer"`) to get a match.

Reply:
- I guess I will revert that change, since the search just searches for common alphabets between the text and the translated text